plugins/modules/packaging/language/ansible_galaxy_install.py

```diff
@@ -0,0 +1,304 @@
+    type: role
+    name: ansistrano.deploy
+    dest: /ansible/roles
+
+- name: Install collections and roles together
+  community.general.ansible_galaxy_install:
+    type: both
+    requirements_file: requirements.yml
+
+- name: Force-install collection community.network at specific version
+  community.general.ansible_galaxy_install:
+    type: collection
+    name: community.network:3.0.2
+    force: true
+
+"""
+
+RETURN = """
+  type:
+    description: The value of the I(type) parameter.
+    type: str
+    returned: always
+  name:
+    description: The value of the I(name) parameter.
+    type: str
+    returned: always
+  dest:
+    description: The value of the I(dest) parameter.
+    type: str
+    returned: always
+  requirements_file:
+    description: The value of the I(requirements_file) parameter.
+    type: str
+    returned: always
+  force:
+    description: The value of the I(force) parameter.
+    type: bool
+    returned: always
+  installed_roles:
+    description:
+    - If I(requirements_file) is specified instead, returns dictionary with all the roles installed per path.
+    - If I(name) is specified, returns that role name and the version installed per path.
+    - "B(Ansible 2.9): Returns empty because C(ansible-galaxy) has no C(list) subcommand."
+    type: dict
+    returned: always when installing roles
+    contains:
+      "<path>":
+        description: Roles and versions for that path.
+        type: dict
+    sample:
+      /home/user42/.ansible/roles:
+        ansistrano.deploy: 3.9.0
+        baztian.xfce: v0.0.3
+      /custom/ansible/roles:
+        ansistrano.deploy: 3.8.0
+  installed_collections:
+    description:
+    - If I(requirements_file) is specified instead, returns dictionary with all the collections installed per path.
+    - If I(name) is specified, returns that collection name and the version installed per path.
+    - "B(Ansible 2.9): Returns empty because C(ansible-galaxy) has no C(list) subcommand."
+    type: dict
+    returned: always when installing collections
+    contains:
+      "<path>":
+        description: Collections and versions for that path
+        type: dict
+    sample:
+      /home/az/.ansible/collections/ansible_collections:
+        community.docker: 1.6.0
+        community.general: 3.0.2
+      /custom/ansible/ansible_collections:
+        community.general: 3.1.0
+  new_collections:
+    description: New collections installed by this module.
+    returned: success
+    type: dict
+    sample:
+      community.general: 3.1.0
+      community.docker: 1.6.1
+  new_roles:
+    description: New roles installed by this module.
+    returned: success
+    type: dict
+    sample:
+      ansistrano.deploy: 3.8.0
+      baztian.xfce: v0.0.3
+"""
+
+import re
+
+from ansible_collections.community.general.plugins.module_utils.module_helper import CmdModuleHelper, ArgFormat
+try:
+    from ansible.module_utils.ansible_release import __version__ as _ansible_version
```

Comment:
This is really, **really** dangerous as there might be no connection *at all* between what can be imported here and what is run with the `ansible-galaxy` CLI tool.

You should really parse the output of `ansible-galaxy --version` instead of doing this.

Reply:
done